includes/Plugin.php

```diff
@@ -400,11 +403,11 @@ public function setWriteApiClient(WriteApi $client): void
     /**
      * Checks if the configured value for the given fulfillment setting is valid.
      *
-     * @param string $optionValue The value of the fulfillment option.
-     * @param string $displayName The name of the setting to be shown in error messages.
+     * @param string|null $optionValue The value of the fulfillment option.
+     * @param string      $displayName The name of the setting to be shown in error messages.
      * @return void
      */
-    private function checkIfFulfillmentSettingExists(string $optionValue, string $displayName): void
+    private function checkIfFulfillmentSettingExists(?string $optionValue, string $displayName): void
     {
```

Comment:
hey @lucapircher quick question, is the ? seen as a hint that the returned value can be null as well? I haven't seen this before 😃

Reply:
Yes exactly, the `?type` indicates that the type is nullable.